config/root-phase0/apiresourceschema-clusterworkspacetypes.tenancy.kcp.dev.yaml

```diff
@@ -49,6 +49,26 @@ spec:
               description: additionalWorkspaceLabels are a set of labels that will
                 be added to a ClusterWorkspace on creation.
               type: object
+            defaultAPIBindings:
+              description: defaultAPIBindings are the APIs to bind during initialization
+                of workspaces created from this type. The APIBinding names will be
+                generated dynamically.
+              items:
+                description: APIExportReference provides the fields necessary to resolve
+                  an APIExport.
+                properties:
+                  name:
```

Comment:
a little odd that an APIBindings list contains exports. Maybe rename `name` to `exportName` ?